.github/workflows/test.yaml

```diff
@@ -15,10 +15,10 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
-      - name: Install Python3.8
+      - name: Install Python3.9
         uses: actions/setup-python@v4
         with:
-          python-version: 3.8
+          python-version: 3.9
```

Comment:
@yadavsahil197 @Abhinav-Naikawadi this change will need to go in for all other repos as well. do we expect any problems because of this?